lib/features/analytics/wiredash/show_feedback_tile.dart

```diff
@@ -0,0 +1,42 @@
+import "package:flutter/material.dart";
+import "package:lottie/lottie.dart";
+import "package:wiredash/wiredash.dart";
+
+import "../../../config/ui_config.dart";
+import "../../../gen/assets.gen.dart";
+import "../../../utils/context_extensions.dart";
+import "../../../widgets/wide_tile_card.dart";
+
+class ShowFeedbackTile extends StatelessWidget {
+  const ShowFeedbackTile({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return WideTileCard(
+      title: context.localize.bug_report_title,
+      subtitle: context.localize.bug_report_subtitle,
+      onTap: () {
+        Wiredash.of(context).show(
+          options: WiredashFeedbackOptions(
+            labels: [
+              Label(
+                id: "label-a9v91o7kdq",
```

Comment:
I'm just wondering, if we can transfer all the labels ids to config file and name them appropriately?

Reply:
sure